addons/mail/models/mail_activity.py

```diff
@@ -127,9 +127,27 @@ def default_get(self, fields):
         'Document Name', compute='_compute_res_name', compute_sudo=True, store=True,
         help="Display name of the related document.", readonly=True)
     # activity
+    @api.model
+    def default_get_activity_type(self):
```

Comment:
According to guidelines (https://www.odoo.com/documentation/13.0/reference/guidelines.html#symbols-and-conventions) this method should be called _default_activity_type_id and located below default_get already present a bit higher in the file. Purpose is to have default, then all fields definitions, then computes, then other methods, to ease file understanding.